src/chm/documents/customactions/wixnetfxextension.html.md

```diff
@@ -7,122 +7,6 @@ after: using_standard_customactions
 
 The [WixNetfxExtension](~/xsd/netfx/index.html) includes a set of custom actions to compile native images using Ngen.exe. For an example, see [How To: NGen managed assemblies during installation](~/howtos/files_and_registry/ngen_managed_assemblies.html).
 
-## PackageGroups
```

Comment:
Yeah, that's a bad name. The content is about the extensions and everything they contain. Probably an artifact of people finding a place that mostly fits. :)